include/cantera/kinetics/Arrhenius.h

```diff
@@ -0,0 +1,339 @@
+// This file is part of Cantera. See License.txt in the top-level directory or
+// at https://cantera.org/license.txt for license and copyright information.
+
+#ifndef CT_ARRHENIUS_H
+#define CT_ARRHENIUS_H
+
+#include "cantera/base/ct_defs.h"
+#include "cantera/base/Units.h"
+#include "cantera/kinetics/ReactionData.h"
+#include "ReactionRate.h"
+#include "MultiRate.h"
+
+namespace Cantera
+{
+
+class AnyValue;
+class AnyMap;
+
+/**
+ *  @defgroup arrheniusGroup  Arrhenius-type Parameterizations
+ *
+ *  This section describes the parameterizations used to describe the standard
+ *  Arrhenius rate parameterization and derived models.
+ *
+ *  @ingroup chemkinetics
+ */
+
+//! Base class for Arrhenius-type Parameterizations
+class ArrheniusBase : public ReactionRate
+{
+public:
+    //! Default constructor.
+    ArrheniusBase();
+
+    //! Constructor.
+    /*!
+     *  @param A  Pre-exponential factor. The unit system is (kmol, m, s); actual units
+     *      depend on the reaction order and the dimensionality (surface or bulk).
+     *  @param b  Temperature exponent (non-dimensional)
+     *  @param Ea  Activation energy in energy units [J/kmol]
+     */
+    ArrheniusBase(double A, double b, double Ea);
+
+    //! Constructor based on AnyValue content
+    ArrheniusBase(const AnyValue& rate,
+                  const UnitSystem& units, const UnitStack& rate_units)
+    {
+        setRateParameters(rate, units, rate_units);
+    }
+
+    //! Perform object setup based on AnyValue node information
+    /*!
+     *  @param node  AnyValue containing rate information
+     *  @param units  Unit system
+     *  @param rate_units  Unit definitions specific to rate information
+     */
+    virtual void setRateParameters(const AnyValue& rate,
+                                   const UnitSystem& units,
+                                   const UnitStack& rate_units);
+    using ReactionRate::setParameters;
```

Comment:
I think this `using` is unnecessary since there aren't any implementations of `setParameters` at the `ArrheniusBase` level.